dashboard/test/models/course_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class CourseTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
I'd say if you redefine `true`, but that doesn't seem to be allowed. Even if I define a method called `true`, it's not called. You could blow up by having another test with the same name or something, but I'm not sure how `assert true` itself could fail. Please do tell :)